maths/Quadratic Equations(Complexes Numbers)

```diff
@@ -0,0 +1,39 @@
+import math
+
+def QuadraticEquation(a,b,c):
+    """
+    Prints the solutions for a quadratic equation, given the numerical coefficients a, b and c,
+    for a*x*x + b*x + c.
+    Ex.: a = 1, b = 3, c = -4
+    Solution1 = 1 and Solution2 = -4
+    """
```

Comment:
Awesome.  Please consider adding these two [__doctest__](https://docs.python.org/3/library/doctest.html) lines at the end of this function's docstring:
```
    >>> QuadraticEquation(5, 6, 1)
    The equation solutions are:  -0.2  and  -1.0
```

Our [automated testing](https://github.com/TheAlgorithms/Python/blob/master/.travis.yml#L14) can then run the first line and raise an error if the result does not match the second line.